packages/eslint-plugin/docs/rules/no-var-requires.md

```diff
@@ -20,6 +20,40 @@ require('foo');
 import foo from 'foo';
 ```
 
+## Options
+
+```ts
+interface options {
+  allowPackageDotJson?: boolean;
+}
+```
+
+- `allowPackageDotJson` : allows to use require for `package.json` alone. default is `false`
+
+configuration
+
+```CJSON
+{
+    // Use type for object definitions
+    "@typescript-eslint/no-var-requires": ["error", { allowPackageDotJson : true } ]
+}
+```
```

Comment:
Delete this - we don't like to include this in rules unless there's something complex
```suggestion
```